nautobot_golden_config/nornir_plays/config_deployment.py

```diff
@@ -27,9 +27,17 @@
 
 
 @close_threaded_db_connections
-def run_deployment(task: Task, logger: logging.Logger, config_plan_qs, deploy_job_result, job_request) -> Result:
+def run_deployment(  # pylint: disable=too-many-arguments, too-many-locals
+    task: Task, logger: logging.Logger, device_to_settings_map, config_plan_qs, deploy_job_result, job_request
+) -> Result:
     """Deploy configurations to device."""
     obj = task.host.data["obj"]
+    settings = device_to_settings_map[obj.id]
+
+    if not settings.deploy_enabled:
```

Comment:
Same as previous. thinking the proper settings verification needs to either happen in the existing verify_settings method and/or earlier in the process.